gui/packages/ubuntupro/lib/pages/subscription_status/subscription_status_page.dart

```diff
@@ -39,19 +39,17 @@ class SubscriptionStatusPage extends StatelessWidget {
                 backgroundColor: YaruColors.red,
               ),
               onPressed: () async {
-                context.read<ValueNotifier<SubscriptionInfo>>().value =
-                    await model.detachPro();
+                await model.detachPro();
```

Comment:
One more thing I just noticed: For both the Store managed and the User managed status pages we need a `Configure Landscape` button that goes back specfically to the Landscape configure page.

Reply:
Should the back button be removed from the page if you get to it that way?

Reply:
For now, I went with adding a new route with a slightly modified Landscape page. If there is a better way to do this, I'd be happy to change the implementation.

Reply:
I'll review it more thoroughly but in the meantime did you consider using `Wizard.of(context).jump()` to go from the status pages back to the Landscape config page?

Reply:
Every time I try to use `jump` I get errors that I believe mean that you cannot jump to previous pages, only next ones. At least, that's what I've experienced in my testing of it.